.travis.yml

```diff
@@ -4,7 +4,7 @@ cache:
     - "$HOME/.m2"
 language: java
 script:
-  - mvn -B -T2 -fae -Djava.awt.headless=true -Dall clean install
+  - mvn -B -fae -Djava.awt.headless=true -Dall clean install
```

Comment:
Interesting changing build settings as part of this pull request - did something happen during a rebase?